interface/forms/newpatient/common.php

```diff
@@ -62,6 +62,7 @@ function sensitivity_compare($a, $b) {
 <?php html_header_show();?>
 <title><?php echo xlt('Patient Encounter'); ?></title>
 
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'];?>/bootstrap-3-3-4/dist/css/bootstrap.css" type="text/css">
```

Comment:
Don't worry, you can focus on the magic and I'll point out the details :)
here's a good codeblock for "bootstrap light light" code:
``````
<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-3-3-4/dist/css/bootstrap.min.css">
<?php if ($_SESSION['language_direction'] == 'rtl') { ?>
    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-rtl-3-3-4/dist/css/bootstrap-rtl.min.css">
<?php } ?>